renderer/test/js_ethereum_provider_browsertest.cc

```diff
@@ -236,6 +236,47 @@ IN_PROC_BROWSER_TEST_F(JSEthereumProviderBrowserTest, AttachIfWalletCreated) {
   EXPECT_EQ(browser()->tab_strip_model()->GetTabCount(), 1);
 }
 
+IN_PROC_BROWSER_TEST_F(JSEthereumProviderBrowserTest, EIP6369) {
```

Comment:
It would be good to add some tests for if another wallet exists to detect regressions in the event that things start interacting oddly with other extenions.